### DISCUSSION TITLE
Support for Wine (or more precisely Proton)

### DISCUSSION THREAD

**Discussion by pavelkux** [naudio/NAudio#1170] [General]
Hello,
I was trying to search for this particular information but did not find anything from recent years. That is why I am asking here.

Is there any chance to have NAudio working for Wine (or more precisely Proton - [](https://en.wikipedia.org/wiki/Proton_(software)) )?
I really need just the basic audio play functionality (with mixer usage).
Thank you for the answer.

**Comment by mdcdi1315** [comment]
There will be no need this to be supported for Wine anymore - NAudio V3 has preliminary support for ALSA, Linux's audio driver. Additionally, there is support for the SoundFile library, that allows reading some common audio formats on all platforms.

There are already V3 NuGet packages that you can consume and test your app.

For all the reasons above, the issue described here is no longer relevant.
